public/scripts/worktrollyUI.js

```diff
@@ -169,6 +169,8 @@ $("#submitCreateNewTask").click(function() {
     });
     $('input').val('');
     $('select').val('');
+    $('textarea').val('');
+    setInterval(setDataIntoDashboard, 1000);
```

Comment:
remove setInterval
instead of this use location.reload() at the place of UI loader.

Reply:
and make changes accordingly.